scripts/dev

```diff
@@ -170,6 +184,31 @@ task :reset => :up do
   sh("docker-compose restart easi", verbose: true) 
 end
 
+namespace :hosts do
+  desc "Verify that hosts for local development are configured"
+  task :check do
+    lookup = `dscacheutil -q host -a name minio`.strip
+    if lookup.size == 0 && ask?("Host 'minio' can't be resolved. Do you want to configure it now? (Y/n)")
+      Rake::Task["hosts:install"].invoke
+
+      Rake::Task["hosts:install"].reenable
+      Rake::Task["hosts:check"].invoke
+    elsif lookup.include?("ip_address: 127.0.0.1")
+      puts "Host 'minio' is resolving correctly."
+    else
+      puts "Host 'minio' resolved, but not to the correct IP:"
+      puts lookup
+      exit(1)
+    end
+  end
+
+  task :install do
+    sh("echo '127.0.0.1 minio' | sudo tee -a /etc/hosts") &&
```

Comment:
This should work on recent macOS machines, but will have to change for older or other OSes.